yt_dlp/extractor/nhk.py

```diff
@@ -539,9 +539,31 @@ class NhkRadiruIE(InfoExtractor):
         },
     }]
 
+    _api_config = None
+
+    def _extract_extended_description(self, episode_id, aa_vinfo2, aa_vinfo3):
+        service, area = aa_vinfo2.split(",")
+
+        if not self._api_config:
+            self._api_config = self._download_xml('https://www.nhk.or.jp/radio/config/config_web.xml',
+                                                  episode_id, 'Downloading API information', fatal=False)
+
+        full_meta = self._download_json(f'https:{self._api_config.find(".//url_program_detail").text}'.format(
+                                        service=service, area=area, dateid=aa_vinfo3),
+                                        episode_id, note='Downloading extended metadata', fatal=False)
```

Comment:
make non-fatal. feel free to try and find better formatting for the `detail_url` lines
```suggestion
        detail_url = try_call(
            lambda: f'https:{self._api_config.find(".//url_program_detail").text}'.format(
                service=service, area=area, dateid=aa_vinfo3))
        if not detail_url:
            return
        full_meta = self._download_json(detail_url, episode_id, 'Downloading extended metadata', fatal=False)
```